nss-day-cohort-48/the-worst-site-ever-fandango
# Finished Click Here and Advert Pages

I have made 2 useless pages that give no information on our team. 

##Steps to Test

1. Pull this branch
2. Open in the browser
3. Ensure "click here to start" leads to jazz page.
4. Ensure Jazz page leads to nonsensical local pages.
